sensor_msgs/PointCloud pc
int32 id_start
int32 id_end